src/index.ts

```diff
@@ -94,13 +98,17 @@ function checkDiskSpace(directoryPath: string, dependencies: Dependencies = {
 	 *
 	 * @param directoryPath - The file/folder path from where we want to know disk space
 	 */
-	function checkWin32(directoryPath: string): Promise<DiskSpace> {
+	async function checkWin32(directoryPath: string): Promise<DiskSpace> {
 		if (directoryPath.charAt(1) !== ':') {
 			return new Promise((resolve, reject) => {
 				reject(new InvalidPathError(`The following path is invalid (should be X:\\...): ${directoryPath}`))
 			})
 		}
 
+		// Some users have WMI disbled, so we attempt to run it before check
+		await cpExecFile('net', ['start', 'Windows Management Instrumentation']).catch(() => undefined)
```

Comment:
When I run this manually, it tells me that the service name is invalid :/

Reply:
Could you try the one I just committed? I think this should work universally.